src/snowflake/snowpark/_internal/ast/utils.py

```diff
@@ -1092,21 +1089,6 @@ def fill_write_file(
             build_expr_from_python_val(t._2, v)
 
 
```

Comment:
what's the reason for removal here? Is this duplicate code?

Reply:
We removed the `PivotValue` entity from the AST due to redundancy. It's simply either a Python constant or a Dataframe, both of which can be captured with a simple `Expr` field instead.